rubber_duck.py

```diff
@@ -132,25 +137,92 @@ async def have_conversation(self, thread_id: int, engine: str, prompt: str, init
                         f'\n{ex}\n'
                         '\n'.join(tb.format_exception(ex))
                     )
+
+                    # For server-side errors
+                    if isinstance(ex,
+                                  (openai.APITimeoutError, openai.InternalServerError, openai.UnprocessableEntityError)):
+                        await self._edit_message(thread_id, self._error_message_id,
+                                                 'I\'m having trouble connecting to the OpenAI servers, '
+                                                 'please open up a separate conversation and try again')
+                    # For client-side errors
+                    elif isinstance(ex, (openai.APIConnectionError, openai.BadRequestError,
+                                         openai.AuthenticationError, openai.ConflictError, openai.ConflictError, openai.NotFoundError,
+                                         openai.RateLimitError)):
+                        user_ids_to_mention = [911012305880358952, 933123843038535741, 1014286006595358791,
```

Comment:
Instead, add a new setting in the config for `admin_ids` that contains a list of ints. Mention those.